ds/set/set_test.go

```diff
@@ -449,63 +454,91 @@ func TestSet_SRem(t *testing.T) {
 
 	assertions.NoError(mySet.SRem(key, []byte("a")))
 	assertions.NoError(mySet.SRem(key, []byte("b")))
+	assertions.NoError(mySet.SRem(key, []byte("c")))
+	assertions.NoError(mySet.SRem(key, make([]byte, 4, 4)))
```

Comment:
see reviewdog tips: 

[golangci] reported by reviewdog 🐶
S1019: should use make([]byte, 4) instead (gosimple)

Raw Output:
ds/set/set_test.go:458:50: S1019: should use make([]byte, 4) instead (gosimple)
	assertions.NoError(mySet.SRem(key, make([]byte, 4, 4)))